plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -1144,13 +1159,120 @@ bool GazeboYarpControlBoardDriver::setPIDsForGroup_IMPEDANCE(std::vector<std::st
     }
 }
 
-bool GazeboYarpControlBoardDriver::setMinMaxImpedance()
+bool GazeboYarpControlBoardDriver::FindMotorControlGroup(yarp::os::Bottle& motorControlGroup_bot) const
```

Comment:
For consistency I would keep method names initials lowercase.

Reply:
Ok